api/backend/reviewers/reviewers_routes.py

```diff
@@ -0,0 +1,89 @@
+########################################################
+# reviewers blueprint of endpoints
+########################################################
+
+from flask import Blueprint
+from flask import request
+from flask import jsonify
+from flask import make_response
+from flask import current_app
+from backend.db_connection import db
+
+#------------------------------------------------------------
+# Create a new Blueprint object, which is a collection of 
+# routes.
+reviewers = Blueprint('reviewers', __name__)
+
+#------------------------------------------------------------
+# Get all the reviewers from the database, package them up,
+# and return them to the client
+@reviewers.route('/reviewers', methods=['GET'])
+def get_reviewers():
+    query = '''
+        SELECT major, name, num_co-ops, year, num_posts, bio 
+        FROM reviewer r
+    '''
+    
+    # get a cursor object from the database~ cursor is like 2d grid that you can iterate over with your code
+    cursor = db.get_db().cursor()
+
+    # use cursor to query the database for a list of products
+    cursor.execute(query)
+
+    # fetch all the data from the cursor
+    # The cursor will return the data as a 
+    # Python Dictionary
+    theData = cursor.fetchall()
+
+    # Create a HTTP Response object and add results of the query to it
+    # after "jasonify"-ing it.
+    response = make_response(jsonify(theData))
+    # set the proper HTTP Status code of 200 (meaning all good)
+    response.status_code = 200
+    # send the response back to the client
+    return response
+
+
+# ------------------------------------------------------------
+# This is a POST route to add a new product.
+# Remember, we are using POST routes to create new entries
+# in the database. 
+@reviewers.route('/reviewers/<reviewerID>', methods=['PUT'])
```

Comment:
Whoops, I didn't notice this when I was dividing the tasks but we should probably have both a POST and PUT route. 
It looks like here you are defining the POST route. 
By convention, if you are adding an object to a db, you don't have a reviewID yet so the route would be '/reviewers'. 
Additionally, the methods should be ['POST'] instead of 'PUT'

Can you also implement a route to update an existing reviewer?
@reviewers.route('/reviewers/<reviewerID>', methods=['PUT'])